Blaze-Javascript restaurant page

ajkacca457 opened the pull request:
- Used webpack to bundle different javascript files.
- all the dom elements are created dynamically using javascript.
- Tab logic used for navigation.

christianotieno commented on README.md:
```diff
@@ -1 +1,58 @@
+
+$ git clone `https://github.com/ajkacca457/Restaurant_page_js.git`
+$ cd Restaurant_page_js
+$ bundle install
```
## Remove 
- [ ] Does this command need to be here? 🤔

christianotieno commented on package.json:
```diff
@@ -0,0 +1,38 @@
+  },
+  "scripts": {
+    "build":"webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
```
### \[OPTIONAL\] Stretch Requirement
- [ ] Do you mind adding the `webpack-dev-server` dependency in our `package.json` and subsequently add it as a start script in here for easier loading and running of the application locally?

christianotieno commented on src/about.js:
```diff
@@ -0,0 +1,48 @@
+import Img1 from './images/img1.png';
+import Img2 from './images/img2.png';
+
+function About() {
```
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.

christianotieno commented on src/contact.js:
```diff
@@ -0,0 +1,82 @@
+import map from './images/map.png';
+
+
+function Contact() {
```
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.

christianotieno commented on src/footer.js:
```diff
@@ -0,0 +1,24 @@
+import whatsapp from './images/whatsapp.png';
+
+
+function Footer() {
```
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.

christianotieno commented on src/homepage.js:
```diff
@@ -0,0 +1,39 @@
+import Patty from './images/wallpaper2.png';
+
+function Homepage() {
```
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.

christianotieno commented on src/index.js:
```diff
@@ -0,0 +1,33 @@
+import Contact from './contact';
+
+
+function init() {
```
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.

christianotieno commented on src/menu.js:
```diff
@@ -0,0 +1,145 @@
+import Banner from './images/foodbanner.jpg';
+import Logo from './images/mainlogo.png';
+
+function Menu() {
```
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.

christianotieno commented on src/navbar.js:
```diff
@@ -0,0 +1,17 @@
+function Navbar() {
```
## Replace function declaration with ES6 syntax
- [ ] Kindly replace this function declaration with the recommended ES6 arrow syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) as is has major benefits like allowing us to implicitly return values when there is nobody block, resulting in shorter and simpler code in some cases.

christianotieno reviewed (changes requested):
### Changes Required ♻️ 
Hello @ajkacca457 👋 

The hard work put into this project is commendable. I love how the codebase is structured. It is very modularised and looks neat. The UI of the page also looks great. Keep it up. 👏  

We, however, do need to take care of some issues before we proceed. Find the suggested changes below 👇 

### Happy coding 💻 
#### Christian Otieno (TSE)

ajkacca457 commented:
@christianotieno  thank you very much for your kind review. I really appreciate your words. I have replaced all function declarations with es6 syntax. I have fixed the mistake in the readME file. Hopefully, my project is now worthy of receiving approval.

christianotieno reviewed (approved):
### Project Approved 🍾 
Hi @ajkacca457  😃 
 ##
This project now looks complete as it has met all the requested changes that were earlier suggested, needed for approval. 👏 

Congratulations on the hard work done with the changes. Good job 👍 
I will go ahead and mark this as complete. Good luck with the next tasks 🥇 
##
### Happy coding 💻 
#### Christian Otieno (TSE)
